solver_setup.py

```diff
@@ -0,0 +1,22 @@
+from distutils.core import setup, Extension
+from Cython.Build import cythonize
+import numpy
+
+setup(ext_modules=cythonize(Extension(
```

Comment:
I as per previous message, it would be better to have a single top level setup.py